{"uuid":"b587a55c-2024-43cc-a523-d66a512ebf6d","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["25170447-7d2e-4b5d-90f3-cab1f8c534f4"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681573065502,"stop":1681573065502},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681573067070,"stop":1681573067071},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"706eed07-64d9-40d7-a211-acf7b0905448-attachment","type":"image/png"},{"name":"Myscreenshot","source":"5bbd47de-b8b6-446d-bdba-cbb7c34efa8c-attachment"}],"parameters":[],"start":1681573077279,"stop":1681573077575},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681573077575,"stop":1681573077576},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681573077577,"stop":1681573077577}],"start":1681573064253,"stop":1681573077578}